cc33conf/suite/trixie/debian/cc33conf.install

```diff
@@ -0,0 +1,7 @@
+cc33xxconf /usr/sbin/cc33conf/
+dictionary.txt /usr/sbin/cc33conf/
+default.conf /usr/sbin/cc33conf/
+README /usr/sbin/cc33conf/
+cc33xx-conf.conf /usr/sbin/cc33conf/
```

Comment:
Why are you putting conf files in sbin?

Reply:
The Yocto recipe, created by the wifi team, does this. So we are doing it as well, since we follow Yocto recipes as much as possible. 
https://git.ti.com/cgit/arago-project/meta-ti/commit/?h=scarthgap&id=bde92b899ec5d6fb7c2585289074990fb25db711

Reply:
They did it wrong in Yocto is no excuse to do it wrong in Debian. Go fix it in the Yocto recipe first if you want to do exactly what the recipe does here.